tests/FakeItEasy.Specs/FakeItEasy.Specs.ruleset

```diff
@@ -4,5 +4,6 @@
   <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
     <Rule Id="CA1003" Action="None" />
     <Rule Id="CA1009" Action="None" />
+    <Rule Id="CA1822" Action="Warning" />

```

Comment:
It's not reduced to warning, it's *raised* to warning. The rule is disabled in the "parent" ruleset (FakeItEasy.Tests.ruleset), and re-enabled in this one.